phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/containers/ContainersController.java

```diff
@@ -51,18 +47,12 @@
     private EngineSettingsManager engineSettingsManager;
     private final EngineToolsManager engineToolsManager;
     private final VerbsManager verbsManager;
-    private Map<String, List<EngineSetting>> engineSettings; // engine settings per engine
-    private Map<String, ApplicationDTO> verbs; // Verbs per engine
-    private Map<String, ApplicationDTO> engineTools; // engine tools per engine
 
     private boolean firstViewSelection = true;
 
-    public ContainersController(ContainersView containersView,
-            ContainersManager containersManager,
-            ContainerEngineController containerEngineController,
-            RepositoryManager repositoryManager,
-            EngineSettingsManager engineSettingsManager,
-            VerbsManager verbsManager,
+    public ContainersController(ContainersView containersView, ContainersManager containersManager,
```

Comment:
I think this was more readable before.

Reply:
Ok, I've reverted the constructor parameter style for the class.